wolfcrypt/src/asn.c

```diff
@@ -24208,6 +24231,16 @@ static int EncodeCertReq(Cert* cert, DerCert* der, RsaKey* rsaKey,
 
 #endif /* WOLFSSL_CERT_EXT */
 
+#ifdef WOLFSSL_CUSTOM_OID
+    /* encode a custom oid and value */
+    /* zero returns, means none set */
+    der->extCustomSz = SetCustomObjectId(cert, der->extCustom,
+        sizeof(der->extCustom), &cert->extCustom);
+    if (der->extCustomSz < 0)
+        return der->extCustomSz;
```

Comment:
Shouldn't this be an error?

Reply:
I replaced with setting `ret` first. A negative return code from `SetCustomObjectId` is an error and was returned, but should not have set `der->extCustomSz`.